doc/topics/releases/3005.rst

```diff
@@ -5,3 +5,117 @@ Salt 3005 Release Notes - Codename Phosphorus
 =============================================
 
 Salt 3005 is currently under development.
+
+
+Python 3.5 and 3.6 Deprecation
+------------------------------
+
+This will be the last release we will support Python versions 3.5 and 3.6.
+In Salt release 3006 we will only support Python versions 3.7 and higher.
+Going forward our policy will be to align with Python's supported versions.
+
```

Comment:
Very minor (not blocking): put a comma after "In Salt release 3006" and after "Going forward".